### DISCUSSION TITLE
Generating "extra" colorscheme files

### DISCUSSION THREAD

**Discussion by jade-tea-h** [EdenEast/nightfox.nvim#384] [Q&A] [answered]
Hi, so I have been fine tuning my configuration of duskfox, just adjusting a couple colors in my neovim configuration. I would like to also get these adjustments for other programs, but as far as I can tell the `misc/extra.lua` file has to be run without any configuration and thus will only use the default colors.

Is there any way to run the generation based off local configuration or should I fork the repo and change the source files?

I love the plugin and especially the library is amazing, super well written! <3

**Comment by EdenEast** [comment, marked as answer]
Sorry I missed this discussion message. The `misc/extra.lua` file can be run like any lua file from neovim. The makefile recipe is a convience function to make sure that I don't commit my personal changes into the repo.

All you have to do is to call nvim without the call to `--clean`.

```bash
nvim --headless -u /path/to/nightfox.nvim/misc/extra.lua
```

there should be a folder called extra in your cwd.
